packages/ckeditor5-html-embed/docs/features/html-embed.md

```diff
@@ -32,6 +32,13 @@ Use the editor below to see the plugin in action.
 
 {@snippet features/html-embed}
 
+## Related features
+
+CKEditor 5 supports a wider range of embed and paste features, including:
```

Comment:
Rule of thumb – put blank lines the way the content is usually rendered on a page. In this case, after a paragraph, there would normally be a margin separating it from the list.

Reply:
Corrected.